rts/Lua/LuaSyncedRead.cpp

```diff
@@ -4445,6 +4445,8 @@ int LuaSyncedRead::GetUnitPosition(lua_State* L)
 /***
  *
  * @function Spring.GetUnitBasePosition
+ * The same as Spring.GetUnitPosition, but without the optional midpoint calculations.
```

Comment:
```
`Spring.GetUnitPosition`
```

Assuming we get reference links to work ideally we wouldn't need `@see`, just make sure references are surrounded by backticks. The `@see` is fine too